public/app/features/alerting/unified/components/rule-editor/ConditionField.tsx

```diff
@@ -1,95 +0,0 @@
-import { css } from '@emotion/css';
```

Comment:
💬 We no longer need this one as conditions are now set from the expression of data query directly.